pkg/chaosctl/cmd/root.go

```diff
@@ -97,12 +100,11 @@ func Execute() {
 		os.Exit(1)
 	}
 
-	// TODO: register recovers
 	recoverCommand, err := NewRecoverCommand(rootLogger.WithName("cmd-recover"), map[string]recover.RecoverBuilder{
-		networkChaos: recover.NewNoopRecover,
-		ioChaos:      recover.NewNoopRecover,
-		httpChaos:    recover.NewNoopRecover,
-		stressChaos:  recover.NewNoopRecover,
+		httpChaos:    recover.PipelineBuilder(httpChaos, recover.CleanProcessRecoverBuilder("tproxy"), recover.HTTPRecover),
```

Comment:
I suggest to move `recover.CleanProcessRecoverBuilder(...).Recover()` into recover.XXXXRecover.
The code of XXXchaos recover is not `HIGH COHESION`.
The readers of code will instinctively think the recover code in side the /recover/httpchaos.go cover all the recover process. So , readers may think the process of recover httpchaos is doing nothing at first. And I think the PipelineBuilder can also be delete here.